router/smoke/openshift_2/test_openshift_02_messaging.py

```diff
@@ -5,7 +5,7 @@
     ReceiverPython, SenderPython, \
     ReceiverNodeJS, SenderNodeJS
 
-MESSAGE_COUNT = {'java': 10, 'python': 100, 'nodejs': 100}
+MESSAGE_COUNT = 10
```

Comment:
Needed to switch this to a simpler approach. This was causing validation issues as well, because when you have a cross client scenario (like java x python), then the java client will expect to see 10 messages while python will expect 100, and so it won't work.

Reply:
Yes.. that makes sense.